src/CLI/Generator.cs

```diff
@@ -13,151 +13,151 @@ namespace CppSharp
                 return false;
             }
 
-            if (string.IsNullOrEmpty(Options.InputLibraryName) && !Options.CheckSymbols)
+            if (string.IsNullOrEmpty(options.InputLibraryName) && !options.CheckSymbols)
             {
                 messages.Add("Input library name not specified and check symbols not enabled. Either set the input library name or the check symbols flag");
                 return false;
             }
 
-            if (string.IsNullOrEmpty(Options.InputLibraryName) && Options.CheckSymbols && Options.Libraries.Count == 0)
+            if (string.IsNullOrEmpty(options.InputLibraryName) && options.CheckSymbols && options.Libraries.Count == 0)
             {
                 messages.Add("Input library name not specified and check symbols is enabled but no libraries were given. Either set the input library name or add at least one library");
                 return false;
             }
             
             StringBuilder tripleBuilder = new StringBuilder();
-            if (Options.Architecture == TargetArchitecture.x64)
+            if (options.Architecture == TargetArchitecture.x64)
                 tripleBuilder.Append("x86_64-");
-            else if(Options.Architecture == TargetArchitecture.x86)
+            else if(options.Architecture == TargetArchitecture.x86)
                 tripleBuilder.Append("i686-");
 
-            if (Options.Platform == TargetPlatform.Windows)
+            if (options.Platform == TargetPlatform.Windows)
             {
                 tripleBuilder.Append("pc-win32-msvc");
-                Abi = CppAbi.Microsoft;
+                abi = CppAbi.Microsoft;
             }
-            else if (Options.Platform == TargetPlatform.MacOS)
+            else if (options.Platform == TargetPlatform.MacOS)
             {
                 tripleBuilder.Append("apple-darwin12.4.0");
-                Abi = CppAbi.Itanium;
+                abi = CppAbi.Itanium;
             }
-            else if (Options.Platform == TargetPlatform.Linux)
+            else if (options.Platform == TargetPlatform.Linux)
             {
                 tripleBuilder.Append("linux-gnu");
-                Abi = CppAbi.Itanium;
+                abi = CppAbi.Itanium;
 
-                if(Options.Cpp11ABI)
+                if(options.Cpp11ABI)
                     tripleBuilder.Append("-cxx11abi");
             }
 
-            Triple = tripleBuilder.ToString();
+            triple = tripleBuilder.ToString();
 
             return true;
         }
 
         public void Setup(Driver driver)
         {
             var parserOptions = driver.ParserOptions;
-            parserOptions.TargetTriple = Triple;
-            parserOptions.Abi = Abi;
+            parserOptions.TargetTriple = triple;
+            parserOptions.Abi = abi;
 
             var options = driver.Options;
-            options.LibraryName = Options.OutputFileName;
+            options.LibraryName = this.options.OutputFileName;
```

Comment:
We only use "this" when we have to make a distinction, for example in a constructor.